### ISSUE TITLE
UPGMA Trees are None-rooted

### ISSUE THREAD

**Issue by yasuiniko** [jeetsukumaran/DendroPy#66]
This might be an intentional feature, but just in case...

The PDM class's upgma_tree() method produces a "None"-rooted tree, similarly to the default Tree class. Is this on purpose? I can't imagine why a user would assume UPGMA creates an unrooted tree.

Best,
Niko

**Comment by jeetsukumaran** [comment]
No, this is a bug. I overlooked setting the rooting state. Will do so. Thanks for pointing it out!

**Comment by yasuiniko** [comment]
Ok, glad to help. Two more things:

1) Is this correct practice for rooting/unrooting trees?
```python
root_status = True # False
my_tree.is_rooted = root_status
my_tree.update_bipartitions()
```

2) The "nj_tree" method also appears to create None-rooted trees.

**Comment by jeetsukumaran** [comment]
Yep! There's more to it under the hood (``is_rooted`` is actually a managed property as is ``is_unrooted``; both ``is_rooted=True`` and ``is_unrooted=False``, and vice versa, update the underlying hidden attribute correctly), but for client code that's all there is to it!

**Comment by yasuiniko** [comment]
Perfect, I'll mark this as closed then.

Thanks for developing this extremely useful library!

**Comment by jeetsukumaran** [comment]
No problem.

Thanks for taking the time to report this and other issues and thereby improve the library!

But I'm re-opening this to keep it open till I actually get around to updating the code to set the rooting.

**Comment by jeetsukumaran** [comment]
Fixed in 285d33b .
